install_deps.cmake

```diff
@@ -110,6 +110,7 @@ if(DEFINED CMAKE_TOOLCHAIN_FILE)
 endif()
 # Initialize directory
 cget(init ${TOOLCHAIN_FLAG} -DCMAKE_INSTALL_RPATH=${PREFIX}/lib ${PARSE_UNPARSED_ARGUMENTS})
+cget(ignore pcre)
```

Comment:
The ignore should be added to the main session in the rbuild.ini file as well. We do ignore it for develop and fin:

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/rbuild.ini#L11
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/rbuild.ini#L21

We just need to add it to main as well, otherwise we will see the same issue when building the package.

Reply:
Review opinion adopted, should be resolved now, please re-review.